src/sentry/api/endpoints/api_tokens.py

```diff
@@ -22,8 +22,11 @@
 from sentry.types.token import AuthTokenType
 
 
-class ApiTokenSerializer(serializers.Serializer):
+class ApiTokenNameSerializer(serializers.Serializer):
     name = CharField(max_length=255, allow_blank=True, required=False)
+
+
+class ApiTokenSerializer(ApiTokenNameSerializer):
```

Comment:
do we need to separate these out? if somebody tries to update another part of the auth token, we should probably give them some kind of error message